### PR TITLE
Create install_Docker_en.md

### PR DESCRIPTION
**Pull request by acosta123**
翻译Install_Docker_en.md英文文档

### CODE REVIEW

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+```
+	#start the previously constructed container
+	docker start [Name of container]
+
+	#enter the started container
+	docker attach [Name of container]
+```
+* If you are new with Docker, please refer ro information on the Internet, such as [Docker Tutorial](http://www.runoob.com/docker/docker-hello-world.html)
+
+## How to Unload
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
## How to uninstall
```

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# **Installation with Docker**
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
# Install on Docker
```
1. 标题不需要\*\*即可粗体化~
2. 暂保持和其他安装标题一致，Docker本身是个环境，paddle是在docker上运行的，宜用on

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+[Docker](https://docs.docker.com/install/) is an open-source application container engine, with which you can seperate the installation and use of PaddlePaddle from the system environment, and share GPU and network resources with the localhost.
+
+## Environment Preparations
+
+- For system types supporting now, please refer to [Installation Instructions](./index_en.html). Using Docker on CentOS 6 is not supported.
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
- For system types supported currently, please refer to [Installation Instructions](./index_en.html). Using Docker on CentOS 6 is not supported.
```

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+	`docker run --name [Name of container] -it -v $PWD:/paddle <imagename> /bin/bash`
+
+	> Parameters in the commands above mean: --name [Name of container]  set the name ofDocker；-it  the parameter means the container has been working interactively with the localhost； -v $PWD:/paddle  appoints to pull the present path(which will unfold to absolute path by PWD variable on Linux) to /paddle catalogue inside the container； `<imagename>` appoints the name of the image needed, and you can check by `docker images` command；/bin/bash  is the command to execute in Docker
+
+Until now, you have installed PaddlePaddle with Docker successfully, for more usage details please refer to [Docker Official Documents](https://docs.docker.com)
+
+<a name="dockers"></a>
+</br></br>
+### **Image Instructions**
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
### Introduction to Images
```

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+1. Pull PaddlePaddle image
+
+	* CPU version PaddlePaddle： `docker pull hub.baidubce.com/paddlepaddle/paddle:[version number]`
+
+	* GPU version PaddlePaddle： `docker pull hub.baidubce.com/paddlepaddle/paddle:[version number]-gpu-cuda9.0-cudnn7`
+
+    Please fill in the version number of PaddlePaddle such as 1.2 after `:`. For more details, please refer to [Image Instructions](#dockers).
+
+2. Construct and enter Docker container
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
2. Construct and enter a Docker container
```

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+	`docker run --name [Name of container] -it -v $PWD:/paddle <imagename> /bin/bash`
+
+	> Parameters in the commands above mean: --name [Name of container]  set the name ofDocker；-it  the parameter means the container has been working interactively with the localhost； -v $PWD:/paddle  appoints to pull the present path(which will unfold to absolute path by PWD variable on Linux) to /paddle catalogue inside the container； `<imagename>` appoints the name of the image needed, and you can check by `docker images` command；/bin/bash  is the command to execute in Docker
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
	> The meanings of the parameters in the command above : --name [Name of container]  sets the name of Docker；-it means the container has been working interactively with the localhost； -v $PWD:/paddle  mounts the present path (PWD will be expanded to the absolute path on Linux) to /paddle directory inside the container； `<imagename>`  specifies the name of the image needed, and you can check it by `docker images` command；/bin/bash  is the command to execute in Docker.
```
"Parameters in the commands above mean"=> use a noun title to introduce the following section.
"挂载"=> mount , mounting
“目录”=> directory
"指定"=> specify .  "appoint" is like to send sb for some job .. "specify" is more "Computer Science " :)

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+Until now, you have installed PaddlePaddle with Docker successfully, for more usage details please refer to [Docker Official Documents](https://docs.docker.com)
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
So far, you have installed PaddlePaddle with Docker successfully, for more usage details please refer to [Docker Official Documents](https://docs.docker.com)
```

"until now" is more frequently used in negative sentences for expressing changes. I think it's more like "Until now I have never been to Disneyland. But now I'm at Disneyland". Anyway, I think "so far" is better :)

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+		<td> hub.baidubce.com/paddlepaddle/paddle:latest-dev </td>
+		<td> newest development environment of PaddlePaddle </td>
+	</tr>
+   </tbody>
+</table>
+</p>
+
+You can find docker images of all versions of PaddlePaddle in [DockerHub](https://hub.docker.com/r/paddlepaddle/paddle/tags/).
+
+### Anouncements
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
### Tips
```
Tips/hints/warning/alerts... Anouncement is perhaps not suitable for "Computer Science" geeks :)

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+* Python version is 2.7 in the image
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
* Python version is 2.7 in the images
```

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+* PaddlePaddle Docker doesn't install `vim` by default in order to decrease the volume, and you can execute `apt-get install -y vim` command in the container for compling codes.
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
* PaddlePaddle Docker doesn't install `vim` by default in order to reduce the volume, and you can install it by executing `apt-get install -y vim` command in the container to edit codes.
```
"decrease" 感觉搭配上，reduce更好一些~
compling =>编译，vim像是sublime这种存在，不是编译器~

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+* When you need to enter Docker container the second time, please use the following commands:
+```
+	#start the previously constructed container
+	docker start [Name of container]
+
+	#enter the started container
+	docker attach [Name of container]
+```
+* If you are new with Docker, please refer ro information on the Internet, such as [Docker Tutorial](http://www.runoob.com/docker/docker-hello-world.html)
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
* If you are new with Docker, please refer to information on the Internet, such as [Docker Tutorial](http://www.runoob.com/docker/docker-hello-world.html) .
```

File: doc/fluid/beginners_guide/install/install_Docker_en.md [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+
+## How to Unload
+
+After entering Docker, execute the following commands:
+
+* ***CPU version PaddlePaddle***: `pip uninstall paddlepaddle`
+
+* ***GPU version PaddlePaddle***: `pip uninstall paddlepaddle-gpu`
+
+or use `docker rm [Name of container]` to delete Docker container directly.
```
Comment (haowang101779990) [inline doc/fluid/beginners_guide/install/install_Docker_en.md]:
```suggestion
or use `docker rm [Name of container]` to delete a Docker container directly.
```

**Review by haowang101779990 — COMMENTED** [review]
Thx for your good translation! 👍   The conversation involves some of my advice to improve it  :)

**Comment by shanyi15** [comment]
建议全部改完统一提一次commit

**Pull request closed without merging**